pkg/operations/remove.go

```diff
@@ -2,6 +2,7 @@ package operations
 
 import (
 	"fmt"
+	"github.com/weaveworks/ignite/pkg/metadata/vmmd"
```

Comment:
this is weird, move to the right location

Reply:
Moved in 650dad1.